datasets/download/downloading-data.qmd

```diff
@@ -64,7 +66,22 @@ where:
 - `<DatasetID>` is the ID of the dataset for which the user wants to download the files
 - `</path/to/output/directory>` is the path to the directory where the files will be downloaded
 - `<filepath_*_to_download>` are the file paths of the files which can be found by listing the files of the dataset as described above
- 
+
+### Download dataset
+
+To download the whole dataset, the user can do it in two steps. First, the user needs to list the files of the dataset and save the file paths to a file
```

Comment:
Maybe ad a note that this is applicable for users of sda-cli version <= 1.3.0

Reply:
did you mean `>=` 1.3.0 ?

Reply:
No, the latest released version is 1.3.0, and this is something we might patch soonish to make it easier for the users until we have time to  properly fix both sda-download and sda-cli. So for a patched version i.e 1.3.1 this might not be needed.

Reply:
It seems to me that the features needed for downloading were merged at `1.3.0`, see [changelog](https://github.com/NBISweden/sda-cli/releases/tag/v0.1.3). So, instead maybe it's better to just write "use version 1.3.0"?

Reply:
It doesn't work since the only usable endpoints right now is:

* /metadata/*
* /s3-encrypted

Reply:
it uses the `s3-encrypted` when supplying a pubkey to download encrypted files. I downloaded the hus dataset yesterday like that.

Reply:
Using the `-dataset` flag? because the code looks like this:
https://github.com/NBISweden/sda-cli/blob/a63d5a3354073a8f1e4498f6fb637cd6b3b44308/download/download.go#L183-L200

Reply:
I added the version in the subheader